src/api/routers/participants/participantsRouter.ts

```diff
@@ -457,7 +458,7 @@ export function createParticipantsRouter() {
 
       const traceId = getTraceId(req);
       const auditTrail = await insertManageApiKeyAuditTrail(
-        participant!,
+        participant,
```

Comment:
sonarlint suggestion

Reply:
Yep - we've confirmed in an earlier check that participant isn't null to the ! isn't needed.